cli/tests/data/compose.users.test.yml

```diff
@@ -1,3 +1,4 @@
+version: '3'
 services:
   astitvasehgal05:
```

Comment:
please change the name to `testuser` and adjust other parts of this compose file accordingly.